portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -13,20 +13,31 @@
 // limitations under the License.
 
 package com.google.sps.servlets;
-
+import java.util.ArrayList;
+import com.google.gson.Gson;
 import java.io.IOException;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
+/* Servlet that returns some example content.*/
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
-
+   
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("Hello Josef!");
+    ArrayList<String> shows = new ArrayList<String>();
+    shows.add("Avatar: The Last Airbender");
+    shows.add("Arrested Development");
+    shows.add("Breaking Bad");
+    response.setContentType("application/json;");
+    response.getWriter().println(convertToJsonUsingGson(shows));
+  }
+  
+  private String convertToJsonUsingGson(ArrayList<String> shows) {
```

Comment:
Likewise as above, good idea to use List

Reply:
Done.